docs/install-manual.md

```diff
@@ -0,0 +1,144 @@
+
+# Create a swarm for development
+
+This section tells you how to create a Docker Swarm on your network to use only for debugging and testing purposes. You can also use this type of installation if you are developing custom applications for Docker Swarm or contributing to it.  
```

Comment:
I haven't worked through the manual process end-to-end yet.  I switched horses to the compose method.  I'll have to update the document for it to vet with me.